tests/containers/helm.pm

```diff
@@ -63,7 +64,15 @@ sub run {
             $provider = publiccloud::aks->new();
         }
         elsif ($k8s_backend eq 'GKE') {
-            die('GKE is not implemented yet');
+            add_suseconnect_product(get_addon_fullname('pcm')) if is_sle;
+            gcloud_install();
+
+            # package needed by init():
+            (is_sle('=15-SP4')) ? 
+               assert_script_run('zypper in -y chrony') : 
```

Comment:
aren't these packages installed by default?

Reply:
No, not by default.
In the test 440 [log](http://marvin5.arch.suse.cz/tests/440/logfile?filename=serial_terminal.txt) if you search  the string `-y ntp` you will see then **ntp** (needed for ntpdate check) not present yet:
```
The following 2 NEW packages are going to be installed:
  libdns_sd ntp
```

Reply:
and why do we need ntp and chrony for google?